repository/dimensions_repository.go

```diff
@@ -0,0 +1,132 @@
+package repository
```

Comment:
I disagree. The models are used by several packages - the Import API client, Kafka consumer, handler & repository IMO they are their own thing.